modules/meta/arrow-meta-prototype/testing-plugin/src/test/resources/Example.kt.meta

```diff
@@ -0,0 +1,17 @@
+import arrow.higherkind
```

Comment:
This file should not be part of VCS as it's rewritten in each compilation by meta

Reply:
It's just used for the comparison as the **expected** result.

The **actual** files are generated in the `/tmp` directory by default:

```
/tmp/Kotlin-Compilation7741532578741152250
├── classes
│   ├── ExampleKt.class
│   ├── ForId2.class
│   ├── ForId2$Companion.class
│   ├── Id2.class
│   └── META-INF
│       └── main.kotlin_module
├── kapt
│   ├── incrementalData
│   ├── kotlinGenerated
│   ├── sources
│   └── stubs
└── sources
    ├── Example.kt
    └── Example.kt.meta
```